src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -537,12 +772,18 @@ Status MediapipeGraphExecutor::infer(const KFSRequest* request, KFSResponse* res
         if (!status.ok()) {
             return status;
         }
+        ++insertedStreamPackets;
+    }
+    if (this->inputNames.size() > insertedStreamPackets) {
+        SPDLOG_DEBUG("Not all input packets created. Expected: {}, Actual: {}. Aborting execution of mediapipe graph: {}",
```

Comment:
3x `{}`, but 2 parameters - will crash at some point. Do we have test executing this code?